src/controllers/registcontroller.js

```diff
@@ -97,6 +108,36 @@ class register {
       });
     }
   }
+
+static async logout(req,res){
+  try {
+    if (req.user.isLoggedIn == true) {
+      await Users.update(
+        { isLoggedIn: false },
+        { where: { email: req.user.email } }
+      );
+      const user = await Users.findOne({
+        where: {
+          email: req.user.email,
+        },
+      }); 
+
+      const isLoggedIn = user.isLoggedIn
+        return res.status(200).json({
+          message: res.__("Logout successfully"),
+          isLoggedIn,
+        });
+      } else {
+        return res.status(200).json({
+          message: res.__("You're not logged in"),
+        });
+      }
+    } catch (error) {
+      console.log(error)
```

Comment:
you can remove this console, maybe you can add to display a message with error.message